docs/config.md

```diff
@@ -4,6 +4,20 @@ The go-ipfs config file is a json document. It is read once at node instantiatio
 either for an offline command, or when starting the daemon. Commands that execute
 on a running daemon do not read the config file at runtime.
 
+#### Profiles
+Configuration profiles allow to tweak configuration quickly. Profiles can be
+applied with `--profile` flag to `ipfs init` or with `ipfs config profile apply`
+command.
+
+- `server` profile
+Recommended for nodes with public IPv4 address, disables host and content
+discovery in local networks.
+
+- `test` profile
+Reduces external interference, useful for running ipfs in test environments.
+Note that with these settings node won't be able to talk to the rest of the
+network without manual bootstrap.
+
 ## Table of Contents
```

Comment:
@whyrusleeping should we document the `badgerds` profile here but label it as experimental?

Reply:
Its somewhat out of scope, but probably a good idea.

Reply:
Out of scope?

I meant to add under the new Profiles section in case that wasn't clear.

Reply:
@kevina ah, gotcha. Yeah, lets get that in here.